tests/DefaultUnleashTest.php

```diff
@@ -826,4 +826,142 @@ private function getInstance(StrategyHandler ...$handlers): DefaultUnleash
             new DefaultVariantHandler(new MurmurHashCalculator())
         );
     }
+
+    public function testGetFeatures()
+    {
+        $instance = $this->getInstance(new DefaultStrategyHandler());
+
+        $this->pushResponse([
+            'version' => 1,
+            'features' => [
+                [
+                    'name' => 'test',
+                    'description' => '',
+                    'enabled' => false,
+                    'impressionData' => true,
+                    'strategies' => [
+                        [
+                            'name' => 'default',
+                        ],
+                    ],
+                ],
+                [
+                    'name' => 'test2',
+                    'description' => '',
+                    'enabled' => true,
+                    'impressionData' => true,
+                    'strategies' => [
+                        [
+                            'name' => 'default',
+                        ],
+                    ],
+                    'variants' => [
+                        [
+                            'name' => 'variant1',
+                            'weight' => 1,
+                            'payload' => [
+                                'type' => 'string',
+                                'value' => 'val1',
+                            ],
+                        ],
+                    ],
+                ],
+            ],
+        ]);
+
+        self::assertNotNull($instance->getFeatures());
```

Comment:
This doesn't make sense, the method can never return null, you should test that the count of features is two and test that individual features are correctly returned.